core/src/main/java/novoda/lib/sqliteprovider/util/DBUtils.java

```diff
@@ -1,93 +1,63 @@
 
 package novoda.lib.sqliteprovider.util;
 
+import android.content.ContentValues;
 import android.database.Cursor;
 import android.database.sqlite.SQLiteDatabase;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Map.Entry;
 
 import novoda.lib.sqliteprovider.sqlite.IDatabaseMetaInfo.SQLiteType;
+import novoda.lib.sqliteprovider.analyzer.Constraint;
+import novoda.lib.sqliteprovider.analyzer.DatabaseAnalyzer;
+import novoda.lib.sqliteprovider.analyzer.StatementGenerator;
+import novoda.rest.database.SQLiteTableCreator;
 
+@Deprecated
```

Comment:
`DatabaseAnalyzer` replaces this. I would have removed this class but it's public.